plugins/browser-plugin-youtube-tracking/src/index.ts

```diff
@@ -66,9 +66,13 @@ function trackEvent(
   });
 }
 
-export function trackingOptionsParser(mediaId: string, conf?: MediaTrackingOptions): TrackingOptions {
+export function trackingOptionsParser(id: string | YT.Player, conf?: MediaTrackingOptions): TrackingOptions {
+  const player: YT.Player | undefined = typeof id === 'string' ? undefined : id;
```

Comment:
I think the compiler understands the type without `: YT.Player | undefined`
Something like: `const player = typeof id === 'string' ? undefined : id;` would suffice